tokio-console/src/config.rs

```diff
@@ -442,6 +502,11 @@ impl Default for Config {
             log_filter: Some(LogFilter(
                 filter::Targets::new().with_default(filter::LevelFilter::OFF),
             )),
+            warnings: vec![
+                KnownWarnings::SelfWakes,
+                KnownWarnings::LostWaker,
+                KnownWarnings::NeverYielded,
+            ],
```

Comment:
this list of default warnings is repeated here and in the CLI arguments. it would be nice to define it in one place, perhaps like this:
```rust
impl KnownWarnings {
    fn default_enabled_warnings() -> Vec<Self> {
           vec![
                KnownWarnings::SelfWakes,
                KnownWarnings::LostWaker,
                KnownWarnings::NeverYielded,
            ]
    }
}
```

then, we could change this to:
```suggestion
            warnings: KnownWarnings::default_enabled_warnings(),
```
and change the `clap(default_values_t = ...)` attribute in the CLI args to:
```rust
    #[clap(default_values_t = KnownWarnings::default_enabled_warnings())]
```

This way, if we change the list of warnings that are enabled by default, such as by removing a default warning or adding a new one, we can update that list in only one place, and both the default config file and the CLI argument defaults will reflect that change.

Reply:
Good suggestion! Addressed.